change/@fluentui-react-breadcrumb-preview-2f782c5e-a8aa-467f-a0cd-fece7f7e03e0.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

```suggestion
  "type": "major",
```

This is actually a breaking change

---

Agree. Why there were no this option while `yarh change` command?